src/Turing.jl

```diff
@@ -20,6 +20,10 @@ using Accessors: Accessors
 import Printf
 import Random
 
+using ADTypes: ADTypes
+
+DEFAULT_ADTYPE = ADTypes.AutoForwardDiff()
```

Comment:
Did you make it non-`const` on purpose?
```suggestion
const DEFAULT_ADTYPE = ADTypes.AutoForwardDiff()
```